sites/www.seafloorexplorer.org.conf

```diff
@@ -8,5 +6,14 @@ server {
         return 301 https://s3.amazonaws.com/$host$request_uri;
     }
 
-    include /etc/nginx/proxy.conf;
+    location / {
+        resolver 8.8.8.8;
+        proxy_set_header       Host zooniversestatic.z13.web.core.windows.net;
+
+        # Serve root index page regardless of what the URI says
+        set $uri_path "www.seafloorexplorer.org/";
+        proxy_pass             https://zooniversestatic.z13.web.core.windows.net/$uri_path;
+
+        proxy_http_version 1.1;
```

Comment:
i'd put this on line 12 to keep it closer to the other proxy request changes before the proxy_pass.

Reply:
Does the order here actually matter or is this just organizational?

Reply:
No - simply cosmetic but helps to quickly understand the configs if the relevant settings are together.